### DISCUSSION TITLE
How to import a Julia module from juliacall?

### DISCUSSION THREAD

**Discussion by nardi** [cjdoris/PythonCall.jl#24] [Q&A]
I was looking at integrating some Julia into an existing Python project, and still trying to decide what the "main" language should be. Both PyCall/PyJulia and this package seem to take Julia as primary, but I like that the reverse direction is quite simple in this package. However, I'm not sure how to do certain things yet, such as import a Julia module from Python. In the docs it says that all functionality is available on the `Main` object, but without string execution functionality similar to `@py`, executing an using/import statement is not really possible, right? The method I found is to implement and use such a string execution function as follows:
```python
def jleval(s, jl=juliacall.Main):
    jl.eval(jl.Meta.parse(s))
def jlusing(m, jl=juliacall.Main):
    jleval(f"using {m};", jl)
```
Is there another better/intended way to accomplish this? And if not, would functionality of this kind be a good addition to the juliacall module? Just to make the available functionality from both sides a bit more symmetric.

**Comment by cjdoris** [comment]
I gave julia modules an extra method `seval` which is like `eval` but for strings. So you can do
```python
juliacall.Main.seval("using Example")
```

The juliacall documentation definitely needs to be made a bit clearer!

**Reply by icweaver** [reply]
I was JUST looking for how to do this, thanks!
